src/modules/Kdump.rb

```diff
@@ -880,6 +881,100 @@ def Summary
       deep_copy(result)
     end
 
+    # Returns available space for Kernel dump according to KDUMP_SAVEDIR option
+    # only local space is evaluated (starts with file://)
+    #
+    # @return [Integer] free space in bytes or nil if filesystem is not local or no
+    #                   packages proposal is made yet
+    def free_space_for_dump
+      kdump_savedir = @KDUMP_SETTINGS.fetch("KDUMP_SAVEDIR", "file:///var/log/dump")
+      log.info "Using savedir #{kdump_savedir}"
+
+      if kdump_savedir.start_with?("/")
+        log.warn "Using old format"
+      elsif kdump_savedir.start_with?("file://")
+        kdump_savedir.sub!(/file:\/\//, "")
+      else
+        log.info "KDUMP_SAVEDIR #{kdump_savedir.inspect} is not local"
+        return nil
+      end
+
+      # unified format of directory
+      kdump_savedir = File.join("/", kdump_savedir)
+
+      partitions_info = SpaceCalculation.GetPartitionInfo()
+      if partitions_info.empty?
+        log.warn "No partitions info available"
+        return nil
+      end
+
+      log.info "Disk usage: #{partitions_info}"
+      # Create a hash of partitions and their free space { partition => free_space, ... }
+      # "name" usually does not start with "/", but does so for root filesystem
+      # File.join ensures that paths do not contain dulplicit "/" characters
```

Comment:
My tests had "var" and "/". On the other hand, I'm using `File.join("/", ...)` which just solves both cases. So, I prefer to be safe and keep it as it is.